### DISCUSSION TITLE
Using my Android tablet as a second monitor

### DISCUSSION THREAD

**Discussion by real-felix** [FreeRDP/FreeRDP#9509] [Q&A] [answered]
Hello, as said in the title, I want to use my tablet as a second monitor. I use GNOME on Wayland. I installed FreeRDP on my tablet.

My questions are:

- Is FreeRDP the right tool for that?
- Do I have to do something on my computer, beside enabling the remote desktop option in the GNOME settings?
- How can I then use my tablet as a monitor? GNOME provides a `ms-rd` address, but I'm not sure what to do with it.

Thanks by advance for your answers.

**Comment by akallabeth** [comment, marked as answer]
if I got it correct: You want to extend your linux/gnome desktop and display part of the session on your android tablet

short answer: no
longer one:
this is not what gnome-remote-desktop and FreeRDP intend to do.
* You only can remotely log in from you android tablet and you have a session adjusted to the device your RDP client  is running on.
* alternatively you can mirror a running gnome session over RDP

both of these do not allow the *extend* you want, only to mirror the primary device or match the device you start the RDP client on

**Reply by real-felix** [reply]
Thank you for your detailed answer.
